### PR TITLE
Torchbench changes for AOT Autograd

### PR DESCRIPTION
**Pull request by anijain2305**
More command line options in torchbench for AOTAutograd

### CODE REVIEW

File: torchdynamo/optimizations/training.py [hunk 1]
```diff
@@ -4,13 +4,18 @@
...
 from torchdynamo.utils import count_calls
+from torchdynamo.utils import counters
 
 from .analysis import has_mutation
 from .backends import BACKENDS
 from .normalize import normalize_ir
 
 log = logging.getLogger(__name__)
 
+counters["aot_autograd"]["total"] = 0
```
Comment (jansel) [inline torchdynamo/optimizations/training.py:15]: Counters are a defaultdict, no need for this -- everything defaults to 0

File: torchbench.py [hunk 2]
```diff
@@ -177,13 +183,17 @@ def print_summary(cls):
...
 
+    @classmethod
+    def aot_summary(cls):
+        return [cls.totals["aot_autograd"]["total"], cls.totals["aot_autograd"]["ok"]]
+
 
-def output_csv(headers, row):
-    assert output_filename
-    existed = os.path.exists(output_filename)
+def output_csv(filename, headers, row):
```
Comment (jansel) [inline torchbench.py:191]: Is this change actually used?  What is motivation?

**Review by jansel — CHANGES_REQUESTED** [review]

File: torchbench.py (hunk 2 above)
Comment (anijain2305) [reply torchbench.py:191]: I am using this later to write *_stats.csv file.

File: torchbench.py [hunk 3]
```diff
@@ -823,6 +866,12 @@ def main():
                 experiment,
                 cos_similarity,
             )
+        stats_file = output_filename.split(".csv")[0] + "_stats.csv"
```
Comment (anijain2305) [inline torchbench.py:869]: @jansel This is the place, I am passing on a new filename. But, yeah, it is not as contained as it should be.

File: torchbench.py (hunk 2 above)
Comment (jansel) [reply torchbench.py:191]: Sounds good, I didn't see that

**Review by jansel — APPROVED** [review]

**Pull request merged**